service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/CaseSubmissionGenerationService.java

```diff
@@ -266,6 +266,13 @@ private String getThresholdDetails(final Grounds grounds) {
             ? grounds.getThresholdDetails() : DEFAULT_STRING;
     }
 
+    private String getSigneeName(final List<Element<Applicant>> applicants) {
```

Comment:
maybe you can move method to caseSubmissionService and then reuse it here and in controller